en_US/security/authz/file.md

```diff
@@ -92,6 +91,5 @@ Means that a client with ID 'light' is **Allowed** to **Subscribe and Publish**
 
 
 ::: tip
-Only a few simple and general rules is contained in acl.conf that make it a system-based ACL principle. If you need to support complex, large amounts of ACL content, you should implement it in an authentication plugin.
-
+Only a few simple and general rules are contained in acl.conf making it a system-based ACL principle. If you need to support complex, large amounts of ACL content, you should implement it in an authentication plugin.
```

Comment:
What exactly is trying to be said here: `Only a few simple and general rules are contained in acl.conf making it a system-based ACL principle`

How does there being only simple rules make it a 'system-based ACL principle'? Is the intention simply to indicate that this is a `simple` ACL and that for anything more complex it should be done via one of the plugins?

Reply:
My attempt to improve it:
```suggestion
The file ACL backend is handy when there are only a small number of general rules, or a small number of username/ClientID based rules.
For large number of per username/CliendID rules, another database ACL backend is recommended to either
as an extension or replacement of the file backend.
```